opcua/server/server.py

```diff
@@ -641,3 +644,42 @@ def set_attribute_value(self, nodeid, datavalue, attr=ua.AttributeIds.Value):
         so it is a little faster
         """
         return self.iserver.set_attribute_value(nodeid, datavalue, attr)
+    
+    def set_user_manager(self, user_manager):
+        """
+        set up a function which that will check for authorize users. Input function takes username
+        and password as paramters and returns True of user is allowed access, False otherwise.
+        """
+        self.user_manager = user_manager
+        self.iserver.user_token_manager = self.user_token_manager
+
+    def user_token_manager(self, token):
```

Comment:
if this method does something (it seems it does ;-)) then I think its name should contain a verb explain what it does: check_user_token(token) for example. But feel free to propose something better